cli/tools/jupyter/server.rs

```diff
@@ -552,16 +552,16 @@ async fn get_jupyter_display(
       "Runtime.callFunctionOn",
       Some(json!({
         "functionDeclaration": r#"async function (object) {
-      if (typeof object[Symbol.for("Jupyter.display")] !== "function") {
-        return null;
-      }
+      const representation = await Deno.jupyter.format(object);
 
-      try {
-        const representation = await object[Symbol.for("Jupyter.display")]();
-        return JSON.stringify(representation);
-      } catch {
-        return null;
-      }
+      // TODO(rgbkrk): Broadcast the execute result directly here
+      // await Deno.jupyter.broadcast("execute_result", {
+      //   data: representation,
+      //   metadata: {},
+      //   execution_count: 999
+      // });
+
+      return JSON.stringify(representation);
```

Comment:
I opted to let the below error handling bubble up any errors.

Reply:
Why is `JSON.stringify` needed here?

Reply:
This part is from https://github.com/denoland/deno/pull/20537 which was to get non-JSON-able entries to be stripped automatically and also (at the time at least) to make it so we didn't have to recursively get all the objects that were in `application/.*json` media types.